packages/uniforms-bootstrap4/src/RadioField.js

```diff
@@ -12,7 +12,8 @@ const Radio = props =>
                 className={classnames(
                     props.inputClassName,
                     'form-check',
-                    `radio${props.inline ? '-inline' : ''}` // bootstrap4 < alpha.6
+                    `radio${props.inline ? '-inline' : ''}`, // bootstrap4 < alpha.6
```

Comment:
IMHO we should get rid of these alpha fixes.

Reply:
done.